engine/src/flutter/lib/web_ui/test/ui/image_golden_test.dart

```diff
@@ -236,6 +236,34 @@ Future<void> testMain() async {
         await matchGoldenFile('${name}_fragment_shader_sampler.png', region: drawRegion);
       });
 
+      test('fragment_shader_sampler with filter quality', () async {
```

Comment:
What I'd like to see in this test is blowing up a tiny pixel art image with the 2 different settings, none and low.  That way we can easily visually validate wether the filter quality is being respected or not.